src/context/Global.tsx

```diff
@@ -110,6 +112,14 @@ const GlobalProvider = (props: { children: any }) => {
         name: "denomination",
         ...stringSerializer,
     });
+    const localeSeparator = (0.1).toLocaleString().charAt(1);
+    const [separator, setSeparator] = makePersisted(
+        createSignal(localeSeparator),
+        {
+            name: "separator",
+            ...stringSerializer,
```

Comment:
`stringSerializer` is used for backwards compatibility. No needed here because it is a new signal